### ISSUE TITLE
Maybe make selectable points more bigger?

### ISSUE THREAD

**Issue by foror** [NoteCAD/NoteCAD#50]
It's too hard to select a some point on a model.

**Comment by Evil-Spirit** [comment]
Yes, this is known issue and this is not because selection radius, it is because query tree structure for 3d model. Not so easy to fix.
